inc/metadata/namespace.php

```diff
@@ -1161,3 +1187,66 @@ function get_in_catalog_option() {
 		return 'pressbooks_publisher_in_catalog';
 	}
 }
+
+/**
+ * This function download the thema subjects from the pressbooks symbionts repo when the book metadata is updated
+ * @param $meta_id
+ * @param $post_id
+ * @param $meta_key
+ * @param $meta_value
+ * @return false
+ */
+function download_thema_lang( $meta_id, $post_id, $meta_key, $meta_value ) {
+	if ( 'pb_language' !== $meta_key || $meta_value === 'en' ) {
+		return false;
+	}
+
+	$thema_lang = $meta_value;
+
+	$thema_lang_baseurl = 'https://raw.githubusercontent.com/pressbooks/symbionts-thema/main/';
+
+	$basepath = WP_CONTENT_DIR . '/uploads/assets/thema/symbionts/';
+
+	$local_file = "{$basepath}{$thema_lang}.json";
+
+	if ( ! is_dir( $basepath ) ) {
+		mkdir( $basepath, 0755, true );
+	}
+
+	if ( ! file_exists( $local_file ) ) {
+		if ( ! function_exists( 'download_url' ) ) {
+			require_once( ABSPATH . 'wp-admin/includes/file.php' );
+		}
+
+		$download_file = "{$thema_lang_baseurl}{$thema_lang}.json";
+
+		$downloaded = download_url( $download_file );
+		if ( is_wp_error( $downloaded ) ) {
+			$_SESSION['pb_errors'][] = sprintf(
+				__( 'Your %1$s thema subjects could not be downloaded from %2$s.', 'pressbooks' ),
```

Comment:
Replace with 'The %1$s Thema subject terms requested for this book could not be downloaded from %2$s. Please report this error to Pressbooks development team using our [bug report form in GitHub](https://github.com/pressbooks/pressbooks/issues/new?assignees=&labels=&template=bug.md&title=)'